azurelinuxagent/ga/update.py

```diff
@@ -284,10 +284,7 @@ def run(self):
                 utc_start = datetime.utcnow()
 
                 last_etag = exthandlers_handler.last_etag
-                try:
-                    exthandlers_handler.run()
-                except RestartError:
```

Comment:
Why remove this? Or do you need to sync?

Reply:
I need to sync.